winml/adapter/winml_adapter_dml.cpp

```diff
@@ -19,10 +19,27 @@
 namespace winmla = Windows::AI::MachineLearning::Adapter;
 
 #ifdef USE_DML
+
+EXTERN_C IMAGE_DOS_HEADER __ImageBase;
+
+static bool IsCurrentModuleInSystem32() {
+  std::string current_module_path;
+  current_module_path.reserve(MAX_PATH);
```

Comment:
Should be resize (data() + size() can be accessed, not + capacity()). Though it should just work as-is